games/static/games/proto.js

```diff
@@ -307,87 +347,146 @@ class Board {
         return !this.squares[this.defenderLocation].isRightBorder()
     }
 
-    shootRocket(e) {
+    shootRocket() {
```

Comment:
this function is quite long - i'd recommend splitting it to smaller functions, or at least add lots of comments who explain why you did some stuff

Reply:
@AvshRLev please do that its important